### PR TITLE
Add support for the Chrome Logger extension

### PR DESCRIPTION
**Pull request by arikfr**
This adds support for logging to Chrome's console using the [Chrome Logger](https://craig.is/writing/chrome-logger) extension. 

This makes it easier to see server side information about the requests we're making (like what SQL queries it generates).

It's only enabled in debug mode.

### CODE REVIEW

File: redash/handlers/chrome_logger.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    chromelogger.info("{} queries executed in {:.2f}ms.".format(queries_count, total_duration*1000))
+
+    chromelogger.group_end("SQL Queries")
+
+
+def chrome_log(response):
+    request_duration = (time.time() - g.start_time) * 1000
+    queries_duration = g.get('queries_duration', 0.0)
+    queries_count = g.get('queries_count', 0.0)
```
Comment (jezdez) [inline redash/handlers/chrome_logger.py]: Maybe use `0` as a fall back for `queries_count`?

File: redash/handlers/chrome_logger.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        response.headers.add(*header)
+
+    return response
+
+
+def init_app(app):
+    if not app.debug:
+        return 
+
+    app.after_request(chrome_log)
```
Comment (jezdez) [inline redash/handlers/chrome_logger.py]: New line at the end of the file?

File: redash/metrics/request.py [hunk 3]
```diff
@@ -32,7 +32,7 @@ def calculate_metrics(response):
                         request_duration,
                         queries_count,
                         queries_duration)
-
+
```
Comment (jezdez) [inline redash/metrics/request.py]: No need for indentation here

**Comment by jezdez** [comment]
BTW, there is also a Firefox extension to enable this for Firefox: https://addons.mozilla.org/en-US/firefox/addon/chromelogger/

File: redash/handlers/chrome_logger.py (hunk 2 above)
Comment (arikfr) [reply redash/handlers/chrome_logger.py]: We really need to setup an auto formatter :-)

**Comment by arikfr** [comment]
@jezdez thanks for the review and the pointer to the Firefox extension. 👍

**Pull request merged**